temporalcli/commandsmd/commands.md

```diff
@@ -4,173 +4,390 @@ Commands for the Temporal CLI.
+Read more: https://docs.temporal.io/cli/activity#complete
 
 #### Options
 
-* `--activity-id` (string) - The Activity to be completed. Required.
-* `--identity` (string) - Identity of user submitting this request.
-* `--result` (string) - The result with which to complete the Activity (JSON). Required.
+* `--activity-id` (string) - Activity `ID` to complete. Required.
+* `--result` (string) - Result JSON for completing the Activity. Required.
+* `--identity` (string) - Identity of the user submitting this request
 
 Includes options set for [workflow reference](#options-set-for-workflow-reference).
 
-### temporal activity fail: Fail an Activity.
+### temporal activity fail: Fail an Activity
+
+Fail an Activity, marking it as having encountered an error during execution.
+Specify the Activity and Workflow Ids.
 
-Fail an Activity.
+```
+temporal activity fail \
+    --activity-id=YourActivityId \
+    --workflow-id=YourWorkflowId
+```
 
-`temporal activity fail --activity-id=MyActivityId --workflow-id=MyWorkflowId`
+Read more: https://docs.temporal.io/cli/activity#fail
 
 #### Options
 
-* `--activity-id` (string) - The Activity to be failed. Required.
-* `--detail` (string) - JSON data describing reason for failing the Activity.
-* `--identity` (string) - Identity of user submitting this request.
-* `--reason` (string) - Reason for failing the Activity.
+* `--activity-id` (string) - `ID` of the Activity to be failed. Required.
+* `--detail` (string) - JSON data describing the reason for failing the Activity
+* `--identity` (string) - Identity of the user submitting this request
+* `--reason` (string) - Reason for failing the Activity
 
 Includes options set for [workflow reference](#options-set-for-workflow-reference).
 
-
 ### temporal batch: Manage Batch Jobs
 
-Batch commands change multiple Workflow Executions.
+A batch job executes a single command affecting multiple Workflow 
+Executions in tandem. Available commands:
+
+* Cancel: Cancel the Workflow Executions specified by the List Filter.
+* Signal: Signal the Workflow Executions specified by the List Filter.
+* Terminate: Terminates the Workflow Executions specified by the List Filter.
+
+You select the Workflow Executions to include and the kind of batch 
+job to apply. For example, cancel all the running 'YourWorkflow' Workflows:
+
+```
+temporal workflow cancel \
+  --query 'ExecutionStatus = "Running" AND WorkflowType="YourWorkflow"' \
+  --reason "Testing"
+ ```
+
+Read more: https://docs.temporal.io/cli/batch
 
 #### Options
 
 Includes options set for [client](#options-set-for-client).
 
-### temporal batch describe: Show Batch Job progress.
+### temporal batch describe: Show Batch Job progress
 
-The temporal batch describe command shows the progress of an ongoing Batch Job.
+Show the progress of an ongoing Batch Job. Pass a valid Job ID
+to return the job's information:
 
-`temporal batch describe --job-id=MyJobId`
+```
+temporal batch describe --job-id=YourJobId
```

Comment:
I believe we should go with space rather than =. Here's why:

- It's the traditional Unix style.
- It's supported more widely and is backwards compatible.
- Consistent with POSIX guidelines for command-line options.

I've added an IN-HOUSE STYLE entry and audited the document.